### ISSUE TITLE
Citation to use this scheduler

### ISSUE THREAD

**Issue by scaomath** [katsura-jp/pytorch-cosine-annealing-with-warmup#5]
@katsura-jp Hi Naoki, I am curious which paper or arXiv to cite if I use this scheduler in my research? Thanks.

**Comment by katsura-jp** [comment]
@scaomath Hi scaomath, this scheduler is based on two papers, SGDR and warmup.

1. SGDR: Stochastic Gradient Descent with Warm Restarts (https://arxiv.org/abs/1608.03983)
2. Accurate, Large Minibatch SGD: Training ImageNet in 1 Hour (https://arxiv.org/abs/1706.02677)

**Comment by scaomath** [comment]
Still SGDR uses warm restarts (from min lr directly to max lr) but your code does a warmup before restarts, which I found in some scenario is better. Goyal et al only does a single warmup phase due to the nature of their problem. I suggest you go making a doi for your repo: https://guides.github.com/activities/citable-code/

Anyway I will cite your code as a github repo manually using a `@misc`.
